include/deal.II/base/mpi_consensus_algorithms.templates.h

```diff
@@ -610,10 +610,18 @@ namespace Utilities
       {
         const auto targets = this->process.compute_targets();
 
+        // The only valid target for a serial program is itself.
         if (targets.size() != 0)
           {
+            Assert(targets.size() == 1,
+                   ExcMessage(
+                     "On a single process, the only valid target "
+                     "is process zero (the process itself), which can only be "
+                     "listed once."));
             AssertDimension(targets[0], 0);
 
+            // This is kind of silly, but let the process send something to
+            // itself.
```

Comment:
Could I ask you to be more serious with your comments! The functions below call the user functions, which do the process-local work (and in particular might have side effects). Feel free to name the vectors differently.

Reply:
This wasn't a comment on your implementation, but on any place that calls the AC code to send a message to itself. It *is* kind of silly to send a message to yourself through the AC machinery :-)